.checkpatch.conf

```diff
@@ -2,7 +2,7 @@
 --no-tree
 --summary-file
 --show-types
---max-line-length=80
+--max-line-length=100
```

Comment:
Does this make it more difficult to integrate with Zephyr? If it does, I would keep the old value and work around it in OpenAMP.

Reply:
No impact on Zephyr integration, the zephyr checkpatch rules are only for the Zephyr Os repository not the modes.